testing/file_test/file_test_base.cpp

```diff
@@ -66,20 +69,25 @@ static auto SplitOutput(llvm::StringRef output)
 // Runs a test and compares output. This keeps output split by line so that
 // issues are a little easier to identify by the different line.
 auto FileTestBase::TestBody() -> void {
-  std::optional<llvm::PrettyStackTraceFormat> stack_trace_entry;
+  std::string test_command;
+  std::optional<llvm::PrettyStackTraceString> stack_trace_entry;
 
   // If we're being run from bazel, provide some assistance for understanding
   // and reproducing failures.
   const char* target = getenv("TEST_TARGET");
   if (target) {
+    constexpr const char* CommandFormat =
```

Comment:
Maybe `llvm::StringLiteral`?

Reply:
This seemed odd to me, but `llvm::formatv` takes a `const char*`, so if I were to use `llvm::StringLiteral` I'd need to `.data()`. I think `llvm::StringLiteral` would only add indirection in this situation.

https://llvm.org/doxygen/FormatVariadic_8h_source.html